internal/neofs/neofs.go

```diff
@@ -212,6 +203,11 @@ func (x *NeoFS) DeleteContainer(ctx context.Context, id cid.ID, token *session.C
 
 // CreateObject implements neofs.NeoFS interface method.
 func (x *NeoFS) CreateObject(ctx context.Context, prm layer.PrmObjectCreate) (oid.ID, error) {
+	ni, err := x.pool.NetworkInfo(ctx, client.PrmNetworkInfo{})
```

Comment:
This is rather expensive to do per-object.

Reply:
Pass in constructor